module/TheWitcherTRPG.js

```diff
@@ -318,4 +318,21 @@ Handlebars.registerHelper({
     or() {
         return Array.prototype.slice.call(arguments, 0, -1).some(Boolean);
     }
-});
\ No newline at end of file
+});
+
+Handlebars.registerHelper('eachLimit', function (context, limit, options) {
```

Comment:
There are so many that they should have their own file. Can you do it in the next update?